front/components/assistant/AssistantDetails.tsx

```diff
@@ -130,13 +238,119 @@ export function AssistantDetails({
       variant="side-sm"
     >
       <div className="flex flex-col gap-5 pt-6 text-sm text-foreground">
-        <DescriptionSection />
-        <AssistantActionsSection
-          agentConfiguration={agentConfiguration}
-          owner={owner}
-        />
-        <InstructionsSection />
+        <TopSection />
+        <TabsSection />
       </div>
     </ElementModal>
   );
 }
+
+function ConfigVersionHeader({
+  agentConfigurationVersion,
+  agentConfiguration,
+  isLatestVersion,
+}: {
+  agentConfigurationVersion: number;
+  agentConfiguration: LightAgentConfigurationType | undefined;
+  isLatestVersion: boolean;
+}) {
+  const getStringRepresentation = useCallback(
```

Comment:
`getAgentConfigurationVersionString` or something like similar. 
The goal of being precise is to make sure in the future newcomers don't get confused trying to use one of these specific functions with generic names for something else 👍